src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
+                    System.out.println("enter the valid path");
+                    return;
+                }
+                String inputfile = args[0];
+                String wordsearch = args[1];
+                File file = new File(inputfile);
+                supportfile(file);
+                contains(file);
+                System.out.println("processing................");
+                String data = null;
+                data = readFile(inputfile);
+                wordsearch(data,wordsearch);
+            }
+            public static String readFile(String fileName)
+            {
+                String data = " ";
```

Comment:
Can this be null and do a null check in main method rather than putting blank space?

Reply:
changed it to null